app/views/profiles/index.html.erb

```diff
@@ -0,0 +1,295 @@
+
+  <main>
+    <div class="container">
```

Comment:
You render this row a lot in different views. Consider consolidating it into a partial.